indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskClient.java

```diff
@@ -129,13 +130,14 @@ public boolean resume(final String id)
         return deserializeMap(responseContent, Map.class, getPartitionType(), getSequenceType());
       } else if (responseStatus.equals(HttpResponseStatus.ACCEPTED)) {
         // The task received the pause request, but its status hasn't been changed yet.
+        RetryPolicy retryPolicy = newRetryPolicy();
```

Comment:
Nit: maybe make this final?

Reply:
done. Though honestly I never found a strong argument in favor of using final. Since most variables here are declared final, changing the code to keep the same style.